app/views/games/show.html.erb

```diff
@@ -1,22 +1,18 @@
-<div id='chessboard'>
+<div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 no-padding">
 	<%= hidden_field :game_id, '', name: nil, id: nil, data: { game_id: @game.id } %>
```

Comment:
I think it would just be easiest to add a 'chessboard' id. It helps describe the div, and can be referenced easily in the future, even if it isn't now.